examples/05_transformers.ipynb

```diff
@@ -0,0 +1,851 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# constructing the transformer\n",
+    "from sktime.transformations.series.summarize import SummaryTransformer\n",
+    "\n",
+    "my_summary_trafo = SummaryTransformer()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "As before, we can fit/apply with `fit`, `transform`, and `fit_transform`.\n",
+    "\n",
+    "`SummaryTransformer` returns primitive features, hence the output will be a `pandas.DataFrame`, each row corresponding to one series in the input. \n",
+    "\n",
+    "If the input is a single series, the output of `transform` and `fit_transform` will be a one-row `DataFrame`, corresponding to mean etc of that one series:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "my_summary_trafo.fit_transform(X_series)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "If the input is a panel, the output of `transform` and `fit_transform` will be a `DataFrame` with as many rows as the `Panel` had series. The row with the corresponding index has mean etc of the series in the panel `X_panel` with the same index:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "my_summary_trafo.fit_transform(X_panel)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### transformers with series/panel output vs primitive output\n",
+    "\n",
+    "Whether a transformer will return primitives, i.e., a `pandas.DataFrame` in `transform`, or time series like objects (`Series` or `Panel`), can be checked by using the `\"scitype:transform-output\"` tag. This is `\"Series\"` for behaviour as in the first example (box-cox), and `\"Primitives\"` for behaviour as in the second example (summarizer):"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "my_boxcox_trafo.get_tag(\"scitype:transform-output\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "my_summary_trafo.get_tag(\"scitype:transform-output\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Use of tags to characterize and search for transformers will be discussed in more detail in section 4."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "NOTE: currently not all transformers are refactored to the full interface that provides vectorization from `Series` to `Panel` data, the above may hence not fully work for all transformers. Contributions to the transformer refactor are very much appreciated."
```

Comment:
this is the first time vectorization is mentioned, so this will confuse some users. We should indicate precisely which parts of the above relied on vectorisation, e.g. we should explain that vectorization is occuring when applying box-cox to a panel.

Reply:
clarified this by removing the reference